packages/status-react/package.json

```diff
@@ -47,6 +47,7 @@
     "@radix-ui/react-visually-hidden": "^0.1.4",
     "@status-im/js": "0.1.0-alpha.2",
     "@stitches/react": "^1.2.8",
+    "date-fns": "^2.28.0",
```

Comment:
`date-fns` is tree-shakable and side-effects free, so it should only pull the particular function. But if we find ourselves needing to do more manipulation with dates, it would make sense to switch to `day.js`.